defs/scenarios/mysql/mysql_router.yaml

```diff
@@ -0,0 +1,26 @@
+checks:
+  using_mysql_router:
+    apt: mysql-router
+  client_ssl_mode:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_mode
+        ops: [[eq, PREFERRED]]
+  client_ssl_cert:
+    config:
+      handler: hotsos.core.plugins.mysql.MySQLRouterConfig
+      assertions:
+        key: client_ssl_cert
+        ops: [[ne, None]]
```

Comment:
i think this needs to be:

[[ne, null]]

This is because Python None translates to null in yaml. if you use None in yaml it actually gives you string "None".